pkg/frontend/adminactions/resources_list_test.go

```diff
@@ -24,111 +24,129 @@ import (
 	mock_env "github.com/Azure/ARO-RP/pkg/util/mocks/env"
 )
 
+func defaultListByResourceGroup(resources *mock_features.MockResourcesClient) {
+	resources.EXPECT().ListByResourceGroup(gomock.Any(), "test-cluster", "", "", nil).Return([]mgmtfeatures.GenericResourceExpanded{
+		{
+			Name: to.StringPtr("vm-1"),
+			ID:   to.StringPtr("/subscriptions/id"),
+			Type: to.StringPtr("Microsoft.Compute/virtualMachines"),
+		},
+		{
+			Name: to.StringPtr("storage"),
+			ID:   to.StringPtr("/subscriptions/id"),
+			Type: to.StringPtr("Microsoft.Storage/storageAccounts"),
+		},
+	}, nil)
+
+	resources.EXPECT().GetByID(gomock.Any(), "/subscriptions/id", azureclient.APIVersion("Microsoft.Storage")).Return(mgmtfeatures.GenericResource{
+		Name:     to.StringPtr("storage"),
+		ID:       to.StringPtr("/subscriptions/id"),
+		Type:     to.StringPtr("Microsoft.Storage/storageAccounts"),
+		Location: to.StringPtr("eastus"),
+	}, nil)
+}
+
+func defaultVirtualMachines(virtualMachines *mock_compute.MockVirtualMachinesClient) {
+	virtualMachines.EXPECT().Get(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Return(mgmtcompute.VirtualMachine{
```

Comment:
This is not your code: you just moved it, but I think there is a same issue as with `diskEncryptionSets.EXPECT().Get(...)`: we need to assert args. Same for `virtualNetworks.EXPECT().Get(...)`.

There are other occurances as well.

Reply:
Thanks for updating old mocks. I see you updated everyitng, but I think this one still needs to be updated.

Reply:
Thanks for updating old mocks. I see you updated everyitng, but I think this one still needs to be updated.